packages/odyssey-react/src/utils/focusHandling.ts

```diff
@@ -40,19 +40,14 @@ function setFocus(elem: OptionalHTMLElement): OptionalHTMLElement {
   const allItems: NodeListOf<HTMLElement> = elem.querySelectorAll(
     FOCUSABLE_ITEMS_SELECTOR
   );
-  const focusableItems = Array.from(allItems).filter((el) => {
-    if (el.tabIndex === -1) {
-      return false;
-    }
-    return true;
-  });
+  const focusableItems = Array.from(allItems).filter((el) => el.tabIndex >= 0);
   // Capture original focused element before setting focus inside modal dialog
   const lastFocusedElement = document.activeElement;
   if (focusableItems.length > 0) {
-    requestAnimationFrame(() => {
+    (() => {
       // Focus on first focusable element inside dialog
       focusableItems[0].focus();
-    });
+    })();
```

Comment:
Can you explain the preference here between requestAnimationFrame and an immediate execution of the function?

Reply:
Oops. This is a mistake. I shouldn't have changed it. Kept it in by mistake.

Reply:
3872c7e75ea017781dc8e53526055f232efcbf0b

Fixed